transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java

```diff
@@ -103,14 +104,20 @@ private void setData(long sqe, long eventId, EventType type, int fd, long buffer
         //Todo cleaner
         //set sqe(submission queue) properties
         PlatformDependent.putByte(sqe + SQE_OP_CODE_FIELD, (byte) type.getOp());
-        PlatformDependent.putByte(sqe + SQE_FLAGS_FIELD, (byte) 0);
         PlatformDependent.putShort(sqe + SQE_IOPRIO_FIELD, (short) 0);
         PlatformDependent.putInt(sqe + SQE_FD_FIELD, fd);
         PlatformDependent.putLong(sqe + SQE_OFFSET_FIELD, offset);
         PlatformDependent.putLong(sqe + SQE_ADDRESS_FIELD, bufferAddress);
         PlatformDependent.putInt(sqe + SQE_LEN_FIELD, length);
         PlatformDependent.putLong(sqe + SQE_USER_DATA_FIELD, eventId);
 
+        //poll<link>read or accept operation
+        if (type == EventType.READ || type == EventType.ACCEPT || type == EventType.POLL_LINK) {
+            PlatformDependent.putByte(sqe + SQE_FLAGS_FIELD, (byte) IOSQE_IO_LINK);
```

Comment:
yeah we need to poll for READ and ACCEPT yeah the basic idea was application to check if there any data here, according to this commit [IORING_FEAT_FAST_POLL](https://git.kernel.dk/cgit/linux-block/commit/?h=for-5.7/io_uring&id=d7718a9d25a61442da8ee8aeeff6a0097f0ccfd6) "that poll<link>other_op is fast as well" which means dont have overhead from async offload, but we still get the POLL event in CQE, it would be quite handy to have this feature `IGNORE_CQE`